website/docs/en/config/module.mdx

```diff
@@ -639,20 +639,24 @@ Inline optimization is applied when the constant value is:
 - `string` with length \<= 6
 
 ```js title="rspack.config.mjs"
+const isProduction = process.env.NODE_ENV === 'production';
+
 export default {
   experiments: {
     inlineConst: true, // This is currently an experimental feature and requires explicit enabling
   },
   module: {
     parser: {
       javascript: {
-        inlineConst: true,
+        inlineConst: isProduction,
       },
     },
   },
 };
 ```
 
+Since this feature relies on export usage information (`optimization.usedExports`), it is recommended to enable it only when `mode = "production"`.
```

Comment:
```suggestion
Since this feature relies on module export usage information ([optimization.usedExports](/config/optimization#optimizationusedexports)), it is recommended to enable it only when `mode = "production"`.
```